broker-plugins/broker-query-engine/src/main/java/org/apache/qpid/server/query/engine/evaluator/settings/QuerySettings.java

```diff
@@ -63,12 +62,12 @@
     /**
      * Maximal amount of queries allowed caching
      */
-    private int _maxQueryCacheSize = HttpPort.DEFAULT_QUERY_ENGINE_CACHE_SIZE;
+    private int _maxQueryCacheSize = 1000;
```

Comment:
The default value constant could be defined in this class (or in some other class in query engine plugin) and reused in HttpManagementConfiguration to avoid duplication of default values.

Reply:
Default query settings values were moved to the separate class DefaultQuerySettings